src/routes/(authenticated)/app/guilds/+page.svelte

```diff
@@ -1,8 +1,161 @@
 <script lang="ts">
+	import { ButtonSize } from '$lib/components/Base/Button/types';
+	
+	import JoyModal from '$lib/components/Advanced/Modal/JoyModal.svelte';
+	import { FontWeight, TextColor, TextSize } from '$lib/components/Base/Text/types'
+	import JoyButton from '$lib/components/Base/Button/JoyButton.svelte'
 	import { page } from '$app/stores'
 	import JoyContainer from '$lib/components/Base/Container/JoyContainer.svelte'
+	import JoyInput from '$lib/components/Base/Input/JoyInput.svelte'
+	import JoyText from '$lib/components/Base/Text/JoyText.svelte'
+	import { TextTag } from '$lib/components/Base/Text/types'
+	import {
+		Border,
+		BorderColor,
+		BorderRounded,
+		ContainerGap,
+		ContainerPadding,
+		Justify,
+	} from '$lib/types'
+	import { ButtonVariant } from '$lib/components/Base/Button'
+	import { AlignItems } from '$lib/types/AlignItems'
+	import { onMount } from 'svelte'
+	import { guilds, guildService } from '$lib/modules/guilds'
+	import { App } from '$lib/modules/app'
+	import { ToastVariant } from '$lib/components/Advanced/Toast/types'
+	import { type newGuildDTO } from '$lib/modules/guilds/types'
+	import { InputVariant } from '$lib/components/Base/Input/types'
+	import JoyTextArea from '$lib/components/Base/Input/JoyTextArea.svelte'
+	import type { PageData } from './$types'
+	import { superForm } from 'sveltekit-superforms'
+	const { loadGuilds } = guildService()
+	const { toast } = App
+
+	
+	export let data : PageData;
+
+	const {createGuild} = guildService()
+
+	
+	
+
+	onMount(async () => {
+		const err = await loadGuilds()
+
+		if (err) {
+			$toast.fire({ message: err.message, variant: ToastVariant.ERROR })
+		}
+	})
+
+
+	let isOpenModal = false;
+
+
+
+	const openModal = () => {
+		isOpenModal = !isOpenModal
+	}
+	
+
+
+	console.log($guilds)
```

Comment:
Hello @arielrecto , please remove the console.logs. You can use code comments instead if you need to take note of something. Thanks